lib/schemaUtils.js

```diff
@@ -2321,7 +2321,12 @@ module.exports = {
     }
 
     // handling authentication here (for http type only)
-    authHelper = this.getAuthHelper(openapi, operation.security);
+    if (options.alwaysInheritAuthentication) {
+      authHelper = this.getAuthHelper(openapi, openapi.security);
```

Comment:
@AndrewGuenther wanted some clarity on why we're setting the value of `authHelper` here